deployment/build_and_release/live/ci/variables.tf

```diff
@@ -0,0 +1,21 @@
+variable "project_id" {
+  description = "The project ID to host the cluster in"
+}
+
+variable "env" {
+  description = "Unique identifier for the env, e.g. ci or prod"
```

Comment:
I don't think we should be defining modules in the live directory. You can see that it's whacky because we need to define an `env` variable here, which is inside the `ci` directory and thus can only really be set to `ci`.